### PR TITLE
Lisää helppo tapa ajastaa toimintoja

### PR DESCRIPTION
**Pull request by M4R774**

### CODE REVIEW

File: bob/main.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env python
 
-import json
```
Comment (taturl) [inline bob/main.py:3]: Json is still used though?

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
+aiocron~=1.8
```
Comment (taturl) [inline requirements.txt:1]: Should add asgiref also to here?

File: bob/scheduler.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+class Scheduler:
+    def __init__(self, updater: Updater):
+        self.updater = updater
+
+    # Nice website for building cron schedules: https://crontab.guru/#0_16_*_*_5
+    # NOTE: Seconds is the LAST element, while minutes is the FIRST
+    # eg. minute hour day(month) month day(week) second
+        @sync_to_async
+        @aiocron.crontab('0 16 * * 5')  # '0 16 * * 5' = “At 16:00 on Friday.”
+        async def yourcouritine():
```
Comment (taturl) [inline bob/scheduler.py:21]: yourcouritine?

File: bob/main.py (hunk 1 above)
Comment (M4R774) [reply bob/main.py:3]: No it is not. It was used for the settings.json, but it is not used anymore.

Comment (taturl) [reply bob/main.py:3]: Line 60, space command. Also weather command has use of json library?

**Review by matiasautio — APPROVED** [review]

**Pull request merged**